### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by ibr5500**
## In this [PR] I have:

> - Implement the design from the sneak peek wireframes.
> - Used previously created methods.
> - Style the app.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.order(:name)
+  end
```
Comment (RolandM99) [inline app/controllers/users_controller.rb]: - [x] #index action for user controller should render all users to its corresponding index view page. Kindly, implement it as below:
```Ruby
 def index
      @users = User.all
  end

```

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @ibr5500, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Gitflow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly, make sure that you add the  column `author_id` in your user table while renaming it. `author_id` should be the foreign key when you'll try to add references to others table columns. For now, when I run the migration, this error below is raised:
image
Kindly, fix it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @RolandM99 for the code review.

**Comment by ibr5500** [comment]
User ids start from 95 to 98

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ibr5500 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used

 # Required Changes ♻️

- [ ]  @ibr5500 great work implementing changes requested but I still get migrations error, kindly ensure you try and recreate a new database then run the migrations to ensure no error goes undetected:
image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @danmainah for the code review.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @ibr5500 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @emmanuellekamwa for the code review.

**Pull request merged**